gql/schema.graphql

```diff
@@ -45,6 +45,8 @@ type Deal {
   PieceCid: String!
   PieceSize: Uint64!
   IsVerified: Boolean!
+  AnnounceToIPNI: Boolean!
+  KeepUnsealedCopy: Boolean!
```

Comment:
Shouldn't the API match the protocol?

Reply:
This is just for display on the front end, and again I think it's clearer to use a positive name for booleans